src/v3/src/transformer/webauthn/transformWebAuthNAuthenticator.test.ts

```diff
@@ -10,33 +10,43 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { ActivationData, ChallengeData, IdxAuthenticator } from '@okta/okta-auth-js';
 import { IDX_STEP } from 'src/constants';
 import { getStubTransactionWithNextStep } from 'src/mocks/utils/utils';
 import {
-  DescriptionElement,
   FormBag,
-  TitleElement,
   UISchemaLayoutType,
-  WebAuthNButtonElement,
   WidgetProps,
 } from 'src/types';
 
 import { transformWebAuthNAuthenticator } from '.';
 
+let mockIsEdgeBrowser = false;
+let mockIsSafariBrowser = false;
```

Comment:
Any notes on why Edge and Safari? Are these proxy indicators for WebAuthn support?
https://webauthn.me/browser-support

Reply:
Oh no, these are just custom items in v2 that we are matching for parity in v3

This is why they perform the safari check, apparently, they dont want to auto trigger the webauthn prompt on safari: https://github.com/okta/okta-signin-widget/blob/0.0/src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js#L129

For edge, they perform the check to add an additional message, here is the message: 
```Note: If you are enrolling a security key and Windows Hello or PIN is enabled, you will need to select 'Cancel' in the prompt before continuing.```